chapters/dashboard/log-stream.md

```diff
@@ -0,0 +1,29 @@
+## LogStream
+
+##### Jina’s LogStream lets you monitor and debug your flows easily. You can audit your running flow live and  filter, group and search based on component (Pod) or log level / message.
+
+![logging](logging.gif)
```

Comment:
could be good to resize this gif. 

The dahsboard README repo does that by adding some html in? idk if thats the proper way to do this

example of that 
`<p align="center">`
`<img src=".github/.README_images/overview.gif?raw=true" alt="log server settings" width="100%">`
`</p>`